### PR TITLE
Add tox gh actions to make sure tests are ok

### PR DESCRIPTION
**Pull request by spapas**

### CODE REVIEW

**Comment by spapas** [comment]
@treyhunner ooh by mistake I made this branch using PR #24 as a base... Let's first merge #24 and then this one (which only adds the tox - github actions functionalit).

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install tox tox-gh-actions
+    - name: Test with tox
+      run: tox
```
Comment (treyhunner) [inline .github/workflows/tests.yml:25]: It looks like your text editor may not be adding newline to the ends of files automatically. If you install an [EditorConfig](https://editorconfig.org/) plugin for your editor, it should auto-add newlines while in this repo ([thanks to this configuration](https://github.com/treyhunner/django-email-log/blob/master/.editorconfig#L8)).

**Review by treyhunner — COMMENTED** [review]
@spapas 👍 I just merged #24

File: .github/workflows/tests.yml (hunk 1 above)
Comment (spapas) [reply .github/workflows/tests.yml]: Yes you are right! I installed the editorconfig plugin for VSCode and it automatically added the newlines to the end :) I added a new commit to this PR.

**Pull request merged**